support-frontend/assets/helpers/user/defaultUserActionFunctions.js

```diff
@@ -62,6 +63,14 @@ function setEmailValidated(emailValidated: boolean): Action {
 }
 
 function setIsReturningContributor(isReturningContributor: boolean): Action {
+  // JTL: We want to send an Ophan event when we recognize a user is a returning contributor and on the landing page
+  const isReturningContributorOnLandingPage = isReturningContributor &&
+    !!document.location.pathname.match(/^https:\/\/support\.\w+\.com\/\w\w\/contribute/);
```

Comment:
Is it guaranteed to end with a slash?
Also, what about campaign pages?

Reply:
I didn't include the 'end' bit of the regex (`$`) so it will match with a campaign code but it's true that it shouldn't include a slash at the end

Reply:
ah yes 👍

Reply:
i just looked at the code - the slash at the end here just indicates it's the end of the regex (JS regexes start and end with `/` -- the character "/" is indicated as `\/`)

Reply:
(so this can be marked as resolved, I believe)